scripts/run-travis-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ev
+npm run lint
+npm run test:coverage
+npm run test:karma
+cat ./test/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+if git log ${TRAVIS_COMMIT_RANGE} | grep -Ei '\[codemod\]'; then
```

Comment:
> I was wondering if including the brackets is too strict. 

I agree, it would be better without the bracket. But that's a minor improvement 😁 .